docs/content/posts/2025-04-30-introducing-the-externalized-kudu-connector.md

```diff
@@ -0,0 +1,89 @@
+---
+title:  "Introducing the externalized Kudu connector"
```

Comment:
I would use "Introducing the Externalized Kudu Connector" to match the subtitle capitalization style.